cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,41 @@
-describe('', () => {
-  before(() => {
+describe('Article flow', () => {
+  let user;
+  let article;
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
-  it('', () => {
-
+  it('Create article', () => {
```

Comment:
check the good test names over here https://mate-academy.github.io/qa-program/checklists/cypress.html and update